II.pass-tests/11-cities-formatter/cities.test.js

```diff
@@ -7,7 +7,7 @@ test('Cities', function () {
         { city: 'Rome', country: 'Italy' },
     ];
 
-    function transform({ city, country }) {
+    function transform( city, country ) {
```

Comment:
is there a reason for this change? Here we are changing the function signature and, instead of passing an object, we are now passing two arguments.
We should not change the tests :)